cmd/metrics-server/app/options/kubelet_client.go

```diff
@@ -72,6 +77,7 @@ func (o *KubeletClientOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.KubeletCAFile, "kubelet-certificate-authority", "", "Path to the CA to use to validate the Kubelet's serving certificates.")
 	fs.StringVar(&o.KubeletClientKeyFile, "kubelet-client-key", "", "Path to a client key file for TLS.")
 	fs.StringVar(&o.KubeletClientCertFile, "kubelet-client-certificate", "", "Path to a client cert file for TLS.")
+	fs.DurationVar(&o.KubeletRequestTimeout, "kubelet-request-timeout", o.KubeletRequestTimeout, "The timeout at request kubelet, must set value less than metric-resolution.")
```

Comment:
Doesn't sound like a correct sentence. Looking at https://kubernetes.io/docs/reference/kubectl/kubectl/ we could write it differently. I would also not mention `metric-resolution` here as `KubeletClientOptions` should not be aware of it.
```suggestion
	fs.DurationVar(&o.KubeletRequestTimeout, "kubelet-request-timeout", o.KubeletRequestTimeout, "The length of time to wait before giving up on a single request to Kubelet. Non-zero values should contain a corresponding time unit (e.g. 1s, 2m, 3h).")
```

Reply:
ok, i got it thank you